lib/hca/user_attributes.rb

```diff
@@ -12,12 +12,15 @@ class HCA::UserAttributes
 
   validates(:first_name, :last_name, :birth_date, :ssn, presence: true)
 
+  # These attributes, along with uuid, are required by mpi/service.
+  # They can be nil as they're not part of the HCA form
+  attr_reader :mhv_icn, :edipi, :gender, :authn_context
+
   def ssn=(new_ssn)
     super(new_ssn&.gsub(/\D/, ''))
   end
 
-  def gender
-    # MVI message_user_attributes expects a gender value but it's not asked on the HCA form
-    nil
+  def uuid
```

Comment:
I think this will need LOA defined as well, won't it?

Reply:
It depends. When [this validation](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/mpi/service.rb#L171) is called `user_identity` is actually an `HCA::UserAttributes` object. That object has its own validations and doesn't currently require `loa`.

So adding `loa` is not necessary to make the tests pass and it's not an attribute the HCA form seems to care about, but I'm not sure if it's needed elsewhere in the application.

Reply:
Ah yes you're right, I forgot that was just wrapped up in the validation part